libsoftwarecontainer/src/gateway/networkgateway.h

```diff
@@ -26,6 +26,65 @@
 #include "jansson.h"
 #include "netlink.h"
 
+
+/**
+ * @brief A rules entry for the treatment of packets.
+ */
+struct IPTableEntry
```

Comment:
The struct is designed for use of NetworkGateway class. Before adding mentioned feature this entry struct was inside of the NetworkGateway class. But i thought it would be better if i collect rule related variables/structure band together. Since the IPTableEntry will hold plain data and all its members ( maybe except insertRule function) are needed to be reachable by NetworkGateway class; thus i believe struct was better option. But it would be better if i move  insertRule funtion to private  members. 

Also i can change this structure to a class if it is believed better way. In this case, a question arises, shall we covert all structures inside IPTableEntry to class as well or not ?